modules/SitecoreImageBuilder/1.0.0/Public/Invoke-Build.ps1

```diff
@@ -108,25 +114,56 @@ function Invoke-Build
                 $previousDigest = (docker image inspect $tag) | ConvertFrom-Json | ForEach-Object { $_.Id }
             }
 
-            # Copy license.xml and any missing source files into build context
+            # Copy any missing source files into build context
             $spec.Sources | ForEach-Object {
                 $sourcePath = $_
 
-                # continue if source file doesn't exist
+                # Continue if source file doesn't exist
                 if (!(Test-Path $sourcePath))
                 {
-                    Write-Warning "Source file '$sourcePath' is missing..."
+                    Write-Warning "Optional source file '$sourcePath' is missing..."
```

Comment:
Why is this an optional source?

Reply:
Before we added the support for "self-packaged WDP's" ie. "Publishing Service", this would throw an error. Now it's a warning only to support adding stuff that not always exist.